src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseExclusions.txt

```diff
@@ -0,0 +1,180 @@
+src/runtime/src/libraries/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs|unknown-license-reference
+src/runtime/src/mono/mono/mini/mini-posix.c|unknown-license-reference
+src/runtime/src/mono/mono/mini/mini-windows.c|unknown-license-reference
+src/runtime/src/native/external/libunwind/doc/libunwind-ia64.*|generic-exception
+src/runtime/src/tests/JIT/Performance/CodeQuality/V8/Crypto/Crypto.cs|unknown-license-reference
+
+# Test data
+src/runtime/src/libraries/System.ServiceModel.Syndication/tests/TestFeeds/AtomFeeds/*.xml
+src/runtime/src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/AttRegexTests.cs|other-permissive
+
+# Reference to a license, not applicable to source
+src/runtime/src/libraries/System.Text.Json/roadmap/images/core-components.txt|unknown-license-reference
+src/runtime/src/libraries/System.Text.Json/roadmap/images/higher-level-components.txt
+
+# Sample data
+src/runtime/src/mono/sample/wasm/browser-webpack/package-lock.json
+
+#
+# source-build-externals
+#
+
+# False positive
+src/source-build-externals/src/abstractions-xunit/README.md|free-unknown
+src/source-build-externals/src/application-insights/NETCORE/ThirdPartyNotices.txt|unknown
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/benchmark/Microsoft.IdentityModel.Benchmarks/CreateTokenTests.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/Microsoft.IdentityModel.JsonWebTokens/JsonClaimValueTypes.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/Microsoft.IdentityModel.Tokens.Saml/Saml/ClaimProperties.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlTokenUtilities.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/Microsoft.IdentityModel.Tokens.Saml/Saml2/ClaimProperties.cs|proprietary-license
+src/source-build-externals/src/azure-activedirectory-identitymodel-extensions-for-dotnet/src/System.IdentityModel.Tokens.Jwt/JsonClaimValueTypes.cs|proprietary-license
+src/source-build-externals/src/humanizer/readme.md|free-unknown
+src/source-build-externals/src/humanizer/NuSpecs/*.nuspec
+src/source-build-externals/src/xunit/README.md|free-unknown
+src/source-build-externals/src/xunit/src/xunit.assert/Asserts/README.md|free-unknown
+
+#
+# source-build-reference-packages
+#
+
+# Contains references to licenses which are not applicable to the source
+src/source-build-reference-packages/src/packageSourceGenerator/PackageSourceGeneratorTask/RewriteNuspec.cs|unknown-license-reference,ms-net-library-2018-11
+
+# API only
+src/source-build-reference-packages/src/referencePackages/**
+src/source-build-reference-packages/src/targetPacks/**
+src/source-build-reference-packages/src/textOnlyPackages/**
```

Comment:
This covers files like `src/source-build-reference-packages/src/textOnlyPackages/src/microsoft.netcore.targets/2.1.0/THIRD-PARTY-NOTICES.TXT` which have a whole set of licenses in them. If these files really are just API only, would it make sense to remove `THIRD-PARTY-NOTICES.TXT` in these SBRP files? Otherwise, distro maintainers will need to look at these files anyway, and they may find issues much later in the release process.

Reply:
@MichaelSimons - What are your thoughts here? Do you think it's correct to remove such files from SBRP? Interestingly, I see that the targeting packs include license files even though the packages contain no implementation. But in general, there are a bunch of [ms-net-library](https://[dotnet.microsoft.com/en-us/dotnet_library_license.htm](https://dotnet.microsoft.com/en-us/dotnet_library_license.htm)) license references in these packages which is not an open-source license.

@omajid - What is your current process for scanning these files? Are you scanning them and excluding them from violations because they are from SBRP?

Reply:
>  Are you scanning them and excluding them from violations because they are from SBRP?

Yes, that's my current approach. Though I think my last scan did not find any non-open source license in there.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/8ecbc21314c8f6d59614f3c42bb37e1aab206b2a